src/confcom/azext_confcom/oras_proxy.py

```diff
@@ -79,7 +79,10 @@ def discover(
                 f"Error pulling the policy fragment from {image}.\n\n"
                 + "Please log into the registry and try again.\n\n"
             )
-        eprint(f"Error retrieving fragments from remote repo: {item.stderr.decode('utf-8')}", exit_code=item.returncode)
+        elif "not found" in item.stderr.decode("utf-8"):
```

Comment:
are the cases of:
1. image is not found in registry
2. image is in registry and there are no fragments
distinct? 

It's unclear from the "not found" if it's talking about fragments or images